api/app/utils/insert_phrases.py

```diff
@@ -0,0 +1,63 @@
+import sys
```

Comment:
Good work! Looks good to me but even I wrote the code you can refer the below




def insert_phrases_from_tsv(filename):
    app = create_app()
    with open(filename, 'r') as file:
        reader = csv.DictReader(file, delimiter='\t')
        header = next(reader)
        primary_lang = Language.query.filter_by(name=header[header.keys()[0]].lower()).first() or Language(name=header[header.keys()[0]].lower())
        secondary_lang = Language.query.filter_by(name=header[header.keys()[1]].lower()).first() or Language(name=header[header.keys()[1]].lower())
        for row in reader:
            primary, secondary = row.values()
            existing_phrase = Phrase.query.filter(
                db.func.lower(Phrase.primary) == primary.lower(),
                db.func.lower(Phrase.secondary) == secondary.lower(),
                Phrase.primary_language == primary_lang,
                Phrase.secondary_language == secondary_lang
            ).first()
            if existing_phrase:
                continue
            phrase = Phrase(
                primary_language=primary_lang,
                secondary_language=secondary_lang,
                primary=primary,
                secondary=secondary
            )
            db.session.add(phrase)
        db.session.commit()


if __name__ == '__main__':
    import sys
    if len(sys.argv) != 2:
        print('Usage: python insert_phrases.py <filename>')
        sys.exit(1)
    filename = sys.argv[1]
    insert_phrases_from_tsv(filename)